vaadin-charts-flow-parent/vaadin-charts-flow-svg-generator/jsdom-exporter.js

```diff
@@ -76,56 +131,63 @@ doc.createElementNS = (ns, tagName) => {
      */
     elem.getBBox = () => {
         let lineWidth = 0,
+            lineHeight = 0,
             width = 0,
-            height = 0;
+            height = 0,
+            newLine = false;
 
         let children = [].slice.call(
             elem.children.length ? elem.children : [elem]
         );
+        children.forEach(child => {
+            if (child.getAttribute('class') === 'highcharts-text-outline') {
+                child.parentNode.removeChild(child);
+            }
+        });
```

Comment:
nit: this code happens twice in this file, it could be refactored into a helper function.

Reply:
Done